plugins/catalog-backend/src/ingestion/LocationAnalyzer.ts

```diff
@@ -24,25 +26,32 @@ import {
 
 export class RepoLocationAnalyzer implements LocationAnalyzer {
   private readonly logger: Logger;
+  private readonly scmIntegrations: ScmIntegrations;
 
-  constructor(logger: Logger) {
+  constructor(logger: Logger, scmIntegrations: ScmIntegrations) {
```

Comment:
the type is `ScmIntegrationsRegistry`, should use that here and above.

Reply:
thanks! updated